tests/unit/amo/components/TestAddonsByAuthorsCard.js

```diff
@@ -213,7 +213,7 @@ describe(__filename, () => {
     expect(root).toHaveClassName('AddonsByAuthorsCard');
   });
 
-  it('should dispatch a fetch action if no add-ons found', () => {
+  it('should always fetch addons by authors', () => {
```

Comment:
I think we should have a comment to not forget about why we want this (with a link to the issue).